pkg/expansion/system_test.go

```diff
@@ -848,7 +1139,7 @@ func sortReusultants(objs []*Resultant) {
 func loadFixture(f string, t *testing.T) *unstructured.Unstructured {
 	obj := make(map[string]interface{})
 	if err := yaml.Unmarshal([]byte(f), obj); err != nil {
-		t.Fatalf("error unmarshaling yaml for fixture: %s", err)
+		t.Fatalf("error unmarshaling yaml for fixture: %s\n%s", err, f)
```

Comment:
```suggestion
		t.Fatalf("%s: error unmarshaling yaml for fixture: %s\s", f, err)
```

?

Also we could rewrite this more as 

```
require.NoError(t, yaml.Unmarshal([]byte(f), obj), "some msg")
```

but I don't mind it either way